package.json

```diff
@@ -1,8 +1,8 @@
 {
```

Comment:
you should add an `.npmignore` that excludes basically everything but the `dist` folder so we don't include stuff like ruby files in the npm package